index.rst

```diff
@@ -5,11 +5,163 @@ LOVE deployment configuration on k8s.
 
-Add content here
-================
+Introduction
+============
+
+The LSST Operators Visualization Environment (LOVE) is the main interface for users to monitor and control the Rubin Observatory Control System.
+This is a React based web application that powered by a Python Django backed.
+:ref:`Below <fig-love-architecture>` is a high-level overview of the different sub-components of the system.
+
+.. figure:: /_static/LOVE_Architecture.png
+  :name: fig-love-architecture
+  :target: ../images/LOVE_Architecture.png
+  :alt: LOVE Architecture
+
+  High-level overview of LOVE's architecture.
+  Arrows indicate the direction of the communication; the dashed lines represents SAL traffic from the observatory middleware whereas solid lines are mainly TCP/IP socket communication.
+
+In the architecture diagram above, the OCS represents the Observatory Control System; these are all the components (CSCs and Scripts) that communicate using the SAL middleware.
+
+Next to the OCS are two LOVE components; the commander and the producer.
+
+The commander is a simple Python web application that consists of a web server that can communicate with the observatory control system through the SAL middleware.
+Clients can request CSC commands to be executed by the commander, which executes them and sends the results back to the client via socket.
+
+Producers, on the other hand, are designed to read data from the OCS (e.g. telemetry and events) through the middleware.
+When they start, a producer will subscribe to the data for an individual component on the system and start listening for new data.
+Separating them by CSC allows load to be distributed between several producers instances.
+Telemetry and events are threated differently by the producers; telemetry data is pooled at a set frequency, while events are listened to asynchronously.
+Furthermore, the producers will continuously attempt to connect to an http server and, once connected, will broadcast any data they receive from the control system.
+
+The manager is a Python Django application.
+At one side they act as a server for the producers to connect to and receive the broadcasted data.
+When the manager receives data from a producer it writes it to a Redis server.
+On the other hand, managers act as a server to broadcast data. 
+When data is written to the Redis server, the manager broadcasts it.
+The manager also broadcast old data from Redis when a new client connects to it and requests it.
+
+Finally, at the user side, there is the frontend application. 
+As mentioned above, the frontend is a JavaScript React web based application.
+The frontend connects to the manager via a socket to receive data from the control system and to the commander to execute commands.
+The connection with the manager is mainly unidirectional, with data flowing from the manager to the frontend.
```

Comment:
I think it might worth noting this because there is currently one endpoint that is being polled every 3 second for each open LOVE browser tab. This means that the managers are also being pounded by HTTP requests increasingly by every new frontend connection.